_includes/_markdown/install-upickle.md

```diff
@@ -0,0 +1,42 @@
+sttp is the HTTP client library of the Scala Toolkit.
```

Comment:
not sure if this is on purpose, but he file name suggests upickle tutorial and it's the same content as in `install-sttp`